internal/bminventory/inventory.go

```diff
@@ -1215,7 +1215,12 @@ func (b *bareMetalInventory) GenerateClusterISOInternal(ctx context.Context, par
 		return nil, common.NewApiError(http.StatusInternalServerError, errors.New(msg))
 	}
 
-	staticNetworkConfig := b.staticNetworkConfig.FormatStaticNetworkConfigForDB(params.ImageCreateParams.StaticNetworkConfig)
```

Comment:
It is V1 code, is it in use still? Do we need to change it?

Reply:
The signature of the function changed.  It will not compile otherwise